vendor kubeadm/v1beta2 rather than directly import this golang file

zhy76 opened the pull request:
Signed-off-by: zhy76 <958474674@qq.com>

<!--  Thanks for submitting a pull request! Here are some tips for you:
1. Please make sure you have read and understood the contributing guidelines: https://github.com/alibaba/blob/master/CONTRIBUTING.md
2. Please make sure the PR has a corresponding issue.
-->

### Describe what this PR does / why we need it
vendor kubeadm/v1beta2 rather than directly import this golang file

### Does this pull request fix one issue?

<!--If that, add "Fixes #xxxx" below in the next line. For example, Fixes #15. Otherwise, add "NONE" -->
Fixes #1743
### Describe how you did it
I deleted the kubeadm/v1beta2 and vendor kubeadm/v1beta2 rather than directly import this golang file by modifing the go.mod,then I modified the other affected files.I want to only vendor the v1beta2 But it can't work.While vendoring,the whole k8s.io/kubernetes package should vendor too.

### Describe how to verify it


### Special notes for reviews

justadogistaken commented:
I notice that "github.com/sealerio/sealer/pkg/runtime/kubernetes/kubeadm/v1beta2" is not deleted. But I guess we could delete it now, because I have the v1beta2 imported.
Is there any problem to delete the v1beta2 in sealer source code?

zhy76 commented:
> I notice that "github.com/sealerio/sealer/pkg/runtime/kubernetes/kubeadm/v1beta2" is not deleted. But I guess we could delete it now, because I have the v1beta2 imported. Is there any problem to delete the v1beta2 in sealer source code?

The “github.com/sealerio/sealer/pkg/runtime/kubernetes/kubeadm/v1beta2” is used by "pkg/clusterfile/decoder.go","pkg/runtime/kubernetes/kubeadm/kubeadm_config.go","pkg/runtime/kubernetes/runtime.go" and "utils/decode.go",and I already changed these files.And I have deleted "github.com/sealerio/sealer/pkg/runtime/kubernetes/kubeadm/v1beta2",it is no problem to delete the v1beta2 in sealer source code.

justadogistaken reviewed (approved):
LGTM
